driver/driver.cpp

```diff
@@ -54,6 +54,11 @@ Routine Description:
     WDFDEVICE hDevice = WdfWdmDeviceGetWdfDeviceHandle(DeviceObject);
     PDEVICE_EXTENSION devExt = FilterGetData(hDevice);
 
+    bool devMatch = true;
+    if (wcsncmp(L"", global.args.device_hw_id, sizeof(global.args.device_hw_id)) != 0) {
```

Comment:
could avoid the call here, just check first char

Reply:
should be fine

Reply:
fixed in new commit